charmdir.go

```diff
@@ -172,10 +174,15 @@ func resolveSymlinkedRoot(rootPath string) (string, error) {
 // ArchiveTo creates a charm file from the charm expanded in dir.
 // By convention a charm archive should have a ".charm" suffix.
 func (dir *CharmDir) ArchiveTo(w io.Writer) error {
-	return writeArchive(w, dir.Path, dir.revision, dir.Meta().Hooks())
+	versionString, err := MaybeGenerateVersionString(dir.Path)
+	if err != nil {
+		logger.Warningf("version file creation/overwrite failed : %v", err)
```

Comment:
change the message and drop the versionString = "" below

Reply:
done